src/platform/EFR32/CHIPDevicePlatformConfig.h

```diff
@@ -94,7 +94,7 @@
 #define CHIP_DEVICE_CONFIG_ENABLE_THREAD_TELEMETRY_FULL 0
 
 #ifndef CHIP_DEVICE_CONFIG_BLE_APP_TASK_NAME
-#define CHIP_DEVICE_CONFIG_BLE_APP_TASK_NAME "BLE_EVENT"
+#define CHIP_DEVICE_CONFIG_BLE_APP_TASK_NAME "Bluetooth event handler"
```

Comment:
So what is the need for longer names here? why 32 bytes for the length?

Reply:
Longer name is required to match the name of the Bluetooth event handler thread, define at line 87 in sl_bt_rtos_adaption.c. As For why 32 bytes and not exactly enough to match this longest name, it is to give us a little bit of flexibility in the event that new and longer name are implemented in gsdk. If we want to have the exact size, we could drop this to 24 (23 char + null termination)

Reply:
Turns out you were right and the truncated name did get a null terminator! New names are now using 12 bytes instead of 24